Sources/KlaviyoForms/InAppForms/IAFPresentationManager.swift

```diff
@@ -28,25 +38,80 @@ class IAFPresentationManager {
         }
     }()
 
-    func setupLifecycleEvents() {
-        lifecycleCancellable = AppLifeCycleEvents.production.lifeCycleEvents()
+    package func handleLifecycleEvent(_ event: String, _ session: String, additionalAction: (() async -> Void)? = nil) async throws {
```

Comment:
```suggestion
    func handleLifecycleEvent(_ event: String, _ session: String, additionalAction: (() async -> Void)? = nil) async throws {
```
Unless the other modules in our Swift Package (i.e., KlaviyoCore or KlaviyoSwift) need access to this code, I think this function can be `internal` instead of `package`. You can leave off the `internal` modifier because that's the default

Reply:
ah got it okay I thought it was necessary for tests but that makes sense

Reply:
the `@testable` tag should take care of exposing those methods for testing